addons/spreadsheet_dashboard_hr_timesheet/data/files/tasks_dashboard.json

```diff
@@ -50,12 +50,12 @@
             "cells": {
                 "A7": {
                     "style": 1,
-                    "content": "[Tasks by Stage](odoo://view/{\"viewType\":\"graph\",\"action\":{\"domain\":[],\"context\":{\"group_by\":[\"stage_id\",\"state\"],\"graph_measure\":\"__count\",\"graph_mode\":\"bar\",\"graph_groupbys\":[\"stage_id\",\"state\"]},\"modelName\":\"report.project.task.user\",\"views\":[[false,\"graph\"],[false,\"pivot\"],[false,\"list\"],[false,\"search\"]]},\"threshold\":0,\"name\":\"Tasks Analysis\"})",
+                    "content": "[Tasks by Stage](odoo://view/{\"viewType\":\"graph\",\"action\":{\"domain\":[],\"context\":{\"group_by\":[\"stage_id\",\"kanban_state\"],\"graph_measure\":\"__count\",\"graph_mode\":\"bar\",\"graph_groupbys\":[\"stage_id\",\"kanban_state\"]},\"modelName\":\"report.project.task.user\",\"views\":[[false,\"graph\"],[false,\"pivot\"],[false,\"list\"],[false,\"search\"]]},\"threshold\":0,\"name\":\"Tasks Analysis\"})",
```

Comment:
I am not sure to understand this change. 🤔

Reply:
I renamed the `state` field in reporting to `kanban_state`, so that it has the same name as for tasks, to avoid inconsistencies and having to edit the search view just to change the field name

Reply:
Oh okay, I see, thanks for the explanation. 🙂